app/Resources/views/admin/blog/show.html.twig

```diff
@@ -5,6 +5,8 @@
 {% block main %}
     <h1>{{ post.title }}</h1>
 
+    {{ include('default/_flash_messages.html.twig') }}
```

Comment:
shoudln't flash messages be displayed in the layout instead of forcing each child page to displat them ?